### PR TITLE
feat(EMS-2120): No PDF - Remove unrequired eligibility questions, re-order flow

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction ✏️ 
This PR updates "insurance eligibility" to have less questions and different ordering, matching the "No PDF" design.

## Resolution ✔️ 
- Remove the following routes, pages and references to:
  - `OTHER_PARTIES_INVOLVED`
  - `LETTER_OF_CREDIT`
  - `PRE_CREDIT_PERIOD`
- Replace `submitAnswersFromBuyerCountryHappyPath` with  `exporterLocation` - this is now the first eligibility form.
- Update E2E tests and UI controllers to redirect to the appropriate eligibility form, starting to align with the "No PDF" design.
- Update the UI's "required data provided" function to match the new flow/ordering.
- Update "start new application" links/flows to match the new flow/ordering.

## Miscellaneous ➕
- Refactor "eligibility" forms  E2E commands to use a generic `selectRadioAndSubmit` function.
- Create cypress commands for all insurance eligibility functions, removing the need to import the functions.
  - Note: buyer country is the only eligibility form that cannot be inside a cypress command. Tried many things, but once the buyer country form selectors/flow are moved into a cypress command, the field selections do not work. Have so far been unable to find a solution for this.
- Update "all sections" E2E test to create an application's eligibility from the user flow, instead of directly in the API.
- Various alignments for consuming cypress `baseUrl`.
- Add missing test coverage for skipping directly to the "insured amount" eligibility form.
- Add `main-application-no-pdf` to GHA workflows.

### CODE REVIEW

File: e2e-tests/commands/insurance/eligibility/forms.js [hunk 1]
```diff
@@ -1,58 +1,54 @@
...
-export const completeStartForm = () => {
-  submitButton().click();
-};
+/**
+ * selectRadioAndSubmit
+ * Select a yes or no radio and submit the form
+ * @param {String} Yes or no string
+ */
+const selectRadioAndSubmit = (answer) => {
+  if (answer === 'yes') {
```
Comment (abhi-markan) [inline e2e-tests/commands/insurance/eligibility/forms.js:9]: Constant?

File: e2e-tests/commands/insurance/eligibility/forms.js [hunk 2]
```diff
@@ -1,58 +1,54 @@
...
+    yesRadio().input().click();
+  }
 
-export const completeCheckIfEligibleForm = () => {
-  submitButton().click();
-};
+  if (answer === 'no') {
```
Comment (abhi-markan) [inline e2e-tests/commands/insurance/eligibility/forms.js:13]: Same as above please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Two comments to addressplease.

File: e2e-tests/commands/insurance/eligibility/forms.js (hunk 2 above)
Comment (ttbarnes) [reply e2e-tests/commands/insurance/eligibility/forms.js:13]: Good shout, have updated.

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**